README.md

```diff
@@ -143,7 +143,7 @@ The audit is provided as development matures. The latest audit report can be fou
 A list of known vulnerabilities can be found here: [Vulnerabilities list #1](https://github.com/valory-xyz/autonolas-registries/blob/main/docs/Vulnerabilities_list%231.pdf)
 
 ## Deployed Protocol
-The list of addresses can be found [here](https://github.com/valory-xyz/autonolas-registries/blob/main/docs/mainnet_addresses.json).
+The list of addresses can be found [here](https://github.com/valory-xyz/autonolas-registries/blob/main/docs/mainnet_addresses).
```

Comment:
Created a folder, all the mainnet json-s will be separate files in it.

Reply:
Ok, then this had to be reflected in all our other smart contract repositories when deployment on multiple chains will happen.  I suggest renaming the folder with 'contract_addresses' and add
- ethereum.json
- goerli.json
Then we will add 
- relevantL2.json

Reply:
I suggest in that folder we keep track of mainnet addresses only, as other network json-s reside in scripts/deployment for dev purposes only.

Reply:
I know that could potentially be found in the deployment scripts, but this would simplify a lot of the addresses research in my opinion. Specifically, from my experience, I can ensure you that this would not require much work from us but simplifies a lot of the team that usually works to prepare protocol documentation since everything can be accessed directly from the main README.

Reply:
Please modify in the next PR